thoughts/plans/html-plan-review-service.html

```
@@ -0,0 +1,810 @@
+      </ul>
+      <h4>Expected files</h4>
+      <ul><li><code>tools/plan-reviewer/src/server/**</code></li><li><code>tools/plan-reviewer/src/cli/**</code></li><li><code>tools/plan-reviewer/src/storage/**</code></li><li><code>tools/plan-reviewer/src/render/**</code></li></ul>
+      <h4>Verify</h4>
+      <pre><code>cd tools/plan-reviewer
+bun run test
+bun run build
+# Start this with the process tool, then run register/index checks in a separate shell; kill the process after verification:
+node dist/bin/plan-review.js serve --db /tmp/plan-reviewer.sqlite --host 0.0.0.0 --port 4317
+node dist/bin/plan-review.js register ../../thoughts/plans/html-plan-review-service.html --url http://127.0.0.1:4317 --json
+node dist/bin/plan-review.js index --url http://127.0.0.1:4317 --json
+curl -fsS http://127.0.0.1:4317/ >/tmp/plan-reviewer-index.html</code></pre>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)